tests/pages/services/ServiceFormModal-cy.js

```diff
@@ -40,10 +40,10 @@ describe("Service Form Modal", function() {
         cy.get(".modal-full-screen").should("to.have.length", 1);
       });
 
-      it("Should Autofocus on the Service ID input field", function() {
+      it.skip("Should Autofocus on the Service ID input field", function() {
```

Comment:
Why are we skipping this and other Tests?

Reply:
Okay got it, https://github.com/cypress-io/cypress/issues/1176 Cypress currently has problems with the focus functionality could you please add a comment here and above all other skipped tests to document why they are skipped.